docs/CodingStyleGuidelines.md

```diff
@@ -98,10 +98,10 @@ Files should contain the following, in order:
 | Location             | Item Type           | Item Name           | File Name              |
```

Comment:
Should the target be `master` or `cv32e40s/dev`?

Reply:
Good point @silabs-robin.  Since this is a documentation item, which affects all projects using core-v-verif, it should probably be `master`.  Having said that, `e40s/dev` will eventually bubble up to master, so I am not strongly opinionated.

Reply:
cv32e40s/dev is fine 👍